src/Microsoft.VisualStudio.Services.Agent/Constants.cs

```diff
@@ -228,5 +229,17 @@ public static class Task
                 public static readonly string DisplayName = "task.displayname";
             }
         }
+
+        public class Release
+        {
+            public class Path
+            {
+                public static readonly string WorkingDirectory = "r";
```

Comment:
Eric, i agree it would have made sense. But we have tweaked our design a little bit as compared to build.
We will not store 'lastReleaseFolderUsed' information anymore. We would compute this number everytime. if i use r1, r2 .. as the folder names then this compuration logic would become a little complex. 
This design also helps to keep the operations idempotent.